test/user_auth_test.go

```diff
@@ -258,6 +252,11 @@ func TestGrantUserDefaultDatabase(t *testing.T) {
 		t.Fatalf("Expected success, got %s", describe(err))
 	}
 
+	// Grant read-only access to default database
```

Comment:
Why moving this code code?
This changes the expected behavior, which may be a problem

Reply:
Yes 3.3.4 changes some behaviour, it mentioned on the CHANGELOG. This was decided to make the use of the LDAP backend easier.

Reply:
Ok. Please make sure this still works on single, resilientsingle & cluster in all permutations.